server/app/rest_api/rest.py

```diff
@@ -147,13 +192,16 @@ class AnnotationsObsAPI(Resource):
         }
     )
     def get(self):
+        content_type = JSON_MIMETYPE
         fields = request.args.getlist("annotation-name", None)
         try:
             annotation_response = current_app.data.annotation({}, "obs", fields)
         except KeyError:
             return make_response(f"Error bad key in {fields}", HTTPStatus.BAD_REQUEST)
         try:
-            return make_response(jsonify(annotation_response), HTTPStatus.OK)
+            return make_response(
+                annotation_response, HTTPStatus.OK, {"Content-Type": content_type}
+            )
         except ValueError as e:
```

Comment:
this restructuring will fail to capture JSON failures (which throw a ValueError).   

I suggest moving the make_response into the first try block, and just catch all of the errors for one try.

Reply:
will do

Reply:
fixed all